.test-defs/PlantTest.yaml

```diff
@@ -0,0 +1,21 @@
+kind: TestDefinition
+metadata:
+  name: plant-test
+spec:
+  owner: DL_5C5BE3E2970B9F404D0E2F50@sap.com # OutputQualification DL
```

Comment:
Can we change this to a google-group address?

Reply:
Talked to @zanetworker - proposing to do that for all the test definitions (having mixed gmail & sap mail) in a separate pr